Meta.Tests.ps1

```diff
@@ -516,12 +516,12 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'Markdown' {
             try
             {
                 Start-Process -FilePath "gulp" -ArgumentList @(
-                        'test-mdsyntax',
-                        '--silent',
-                        '--rootpath',
-                        $repoRootPath,
-                        '--dscresourcespath',
-                        $dscResourcesFolderFilePath) `
+                    'test-mdsyntax',
+                    '--silent',
+                    '--rootpath',
+                    $repoRootPath,
+                    '--dscresourcespath',
+                    $dscResourcesFolderFilePath) `
```

Comment:
I think this should be indented in even though VS code doesn't indent them.  I don't find it to be a big issue to check it in this way, but I know the VS Code PowerShell reformatter is not 100% yet.  So we should not blindly do what it say.